PostPairsTrading.md

```diff
@@ -0,0 +1,260 @@
+
+> 💡 This strategy possesses intriguing financial characteristics: our **profitability remains unaffected by the broader market trends**, as our focus lies solely on the disparity between the two assets. It's about relative movements rather than absolute ones; we're indifferent to whether prices are rising or falling. This quality defines it as a **neutral market strategy**.
+
+## Spreading spreads
+
+Indeed, just subtracting the prices of two assets, as in: $priceY−priceX$ may not provide a clear understanding of their relationship. Let's illustrate this with an example:
+
+Consider the following series:
+
+---
+
+```q
+priceX: 5 10 7 4 8
+
+priceY: 23 30 25 30 35
+
+spreads: priceY - priceX // = 18 20 18 26 27
+```
+
+> 💡 As you can see and verify through Q/kdb+ array properties, spreads can be calculated by simply computing the difference between two vectors without the need for special functions or loops.
+
+---
+
+〽 **These spread values don't offer much insight** into the relationship between the two assets. Are both assets increasing? Are they moving in opposite directions? It's unclear from these numbers alone.
+
+Let's consider **using logarithms**, as they possess favorable properties for our pricing model. They inherently prevent negative values and tend to approach zero:
+
+--- 
+
+```q
+log priceX = 1.609438 2.302585 1.94591 1.386294 2.079442
+
+log priceY = 3.135494 3.401197 3.218876 3.401197 3.555348
+
+spreads: log priceY - log priceX // = 1.526056 1.098612 1.272966 2.014903 1.475907
+```
+
+---
+
+We're making progress, as we observe **numbers now fluctuating within much smaller ranges**. However, we're still missing a clear understanding of the underlying relationship. While we've normalized the data using logarithms, we now need to align their discrepancies to a single asset. 
+
+Since both assets are related, **we can leverage linear regression** to our advantage. This enables us to simplify our spreads effectively. So, we'll conduct a basic linear regression analysis using historical data to discern the disparity between them:
+
+--- 
+
+$spread = log(priceY) - (beta * log(priceX)+alpha)$
+
+```q
+historical_data_priceX: 7 10 6 5 8
+
+historical_data_priceY: 23 25 16 20 15
+
+beta: 0.2679227
+
+alpha: 2.444817
+```
+
+---
+
+We've already calculated the alpha and beta using the logarithmic values of our historical data (since we don't have prior knowledge of the real-time price values for priceX and priceY). Now, all that remains is to apply the previous formula to derive our spreads:
+
+---
+
+$spreads$ = -0.1493929 0.0451223 -0.08835117 0.0451223 0.1579725

```

Comment:
I think this is quite hard to understand (there are so much decimals), maybe you dont need to wirte the entire formula, maybe you can just say that just evaluating you get that result